app/containers/CreateWallet/CreateWallet.jsx

```diff
@@ -15,73 +21,108 @@ type State = {
   passphrase: string,
   passphrase2: string,
   wif: string,
+  option: string,
```

Comment:
Maybe change this to `oneOf(['CREATE', 'IMPORT'])` to be more explicit?

Reply:
👍